spec/unit/mutest/integration_spec.rb

```diff
@@ -11,14 +11,15 @@
   end
 
   describe '.setup' do
-    subject { described_class.setup(kernel, name) }
+    subject { described_class.setup(requirer, name) }
 
-    let(:name)   { 'null'               }
-    let(:kernel) { class_double(Kernel) }
+    let(:name) { 'null' }
+    let(:requirer) { Mutest::Requirer.new }
 
     before do
-      expect(kernel).to receive(:require)
+      expect(requirer).to receive(:require)
         .with('mutest/integration/null')
+        .and_call_original
```

Comment:
I thought about this some more and I think the best solution is to split the tests: Have one test that tests the process (basically how it was), and one test that checks existence of the file for all supported integrations.